pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -93,6 +96,9 @@ const (
 	// ProxyInboundListenPort is the port on which all inbound traffic to the pod/vm will be captured to
 	// TODO: allow configuration through mesh config
 	ProxyInboundListenPort = 15006
+
```

Comment:
This is not how you check for proxy version. Search the code for examples of util.IsProxyVersion... so you would have a check for 1.3.x